plugins/outputs/nats/sample.conf

```
@@ -66,3 +66,8 @@
     # allow_rollup_hdrs = false
     # allow_direct = true
     # mirror_direct = false
+
+    ## Set to true if the stream is managed externally and Telegraf should not attempt
+    ## to create or update it. When true, Telegraf will connect to the existing stream
+    ## and fail if it does not exist.
+    # external_stream_config = false
```

If I understand this whole PR correctly that flag would prevent the Telegraf plugin from creating the stream but assuming it's already there, doesn't it? If so why not name it as such as your flag name suggests something more... How about

```suggestion
    ## Disable creating the stream but assume the stream is managed externally
    ## and already exists. This will make the plugin fail if the steam does not exist.
    # disable_stream_creation = false
```